client/state/ui/preview/actions.js

```diff
@@ -30,19 +20,6 @@ export function clearPreviewUrl() {
 	};
 }
 
-export function setPreviewType( previewType ) {
```

Comment:
This is still used in two places:

- `client/components/web-preview/with-preview-props.jsx`
- `client/extensions/zoninator/components/forms/zone-content-form/post-card.jsx`

Reply:
Oh, I totally missed the Zoninator dangling import. I'd expect that the Webpack build totally fails on a missing import instead of just a warning. Fixed.

The `WithPreviewProps` component is not used anywhere, so I removed it in https://github.com/Automattic/wp-calypso/pull/23613/commits/906eb996dceb80f08a4f0e73e04dbd6c8cfe78c3